packages/cozy-konnector-libs/src/index.js

```diff
@@ -37,6 +37,7 @@ module.exports = {
   normalizeFilename: require('./libs/normalizeFilename'),
   utils: require('./libs/utils'),
   solveCaptcha: require('./libs/solveCaptcha'),
```

Comment:
Do we still need to expose this `solveCaptcha`? Shoudn't we use only `solveCaptchaWithFullResponse` ?

Also, I don't understand the `withFullResponse` since we still return `solveWithAntiCaptcha`. And this `fullResponse` mean?

Reply:
In that case, why not editing the signature of the `solveCaptcha` method to something like: 

```
solveCaptcha(params, fullResponse = false)
```

By default, solveCaptcha will continue to return the same thing as before, but you can get the fullResponse by setting this arg to true. Like that : only one method to expose. 

If the second arg is not a good option, you can also pass the information to the first param object. 

I prefer to have as small an API surface as possible instead of creating new methods. It'll be easier for a dev to know what to do. (and for us to maintain)

Reply:
We thought about adding an argument.
That's was the first choice but after talking with @doubleface with think it's cleaner to have 2 method with different kind of return.

We need to talk about it.

Reply:
I found it better to enforce the right api, which could return an object with possibly  multiple keys instead of a string.

Using fullResponse parameter works but enforces an api which can return different types of responses depending on parameters, which is less readable to my opinion.

Reply:
This is where inferred types are great. Depending of what args you pass, you get the right return type. Our API is way too big today, we expose too more functions and it's hard to know.

Having a method that return <string, string[]> is not bad, is it?

Reply:
It looked to me as a bad practice to return different types of data depending on the parameters.
But let's do this. And it will be simpler to implement.